src/openrct2-ui/windows/ViewClipping.cpp

```diff
@@ -132,7 +132,7 @@ class ViewClippingWindow final : public Window
                     tool_cancel();
                 }
                 gClipSelectionA = { 0, 0 };
-                gClipSelectionB = { MAXIMUM_MAP_SIZE_BIG - 1, MAXIMUM_MAP_SIZE_BIG - 1 };
+                gClipSelectionB = gMapSize.ToCoordsXY();
```

Comment:
Initializing it to the map size will give undesired effects when the map is resized. For this reason, it was initialized to the max value it can have.

Reply:
Instead of dealing with these min/max settings, we could make `gClipSelectionA/B` an `std::optional<std::pair<CoordsXY, CoordsXY>>`, so we know when it is set and can clear it explicitly, instead of assigning magic values to it.